### PR TITLE
feat: participants summary

### PR DESCRIPTION
**Pull request by ReedSoftware**

### CODE REVIEW

File: server/routes/journeys/record-review/participants-summary/view.njk [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends "partials/layout.njk" %}
+
+{% set fullWidth = attendees.length %}
+{% set pageTitle = "Participant and contribution details - Record a CSIP review" %}
+
+{% block innerContent %}
+    {% include 'base/components/review-caption.njk' %}
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-7">Log participants and contributions to the review</h1>
```
Comment (bill-moj) [inline server/routes/journeys/record-review/participants-summary/view.njk:11]: this heading needs to be limited to two-third width

File: server/routes/journeys/record-review/participants-summary/view.njk [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+    {% for attendee in attendees %}
+        {{ govukSummaryList({
+            card: {
+                title: { text: "Participant: " + attendee.name },
+                actions: {
+                    items: [
+                        {
+                            href: 'delete-participant/' + attendee.attendeeUuid,
```
Comment (bill-moj) [inline server/routes/journeys/record-review/participants-summary/view.njk:20]: attendees don't have uuid yet, we can only link to them via index

same with the urls below

File: server/routes/journeys/record-review/participants-summary/view.njk [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                            text: "Delete",
+                            visuallyHiddenText: "the information about the participant (" + attendee.name + ")",
```
Comment (bill-moj) [inline server/routes/journeys/record-review/participants-summary/view.njk:22]: the card title is automatically appended to the visually hidden text.
ie, ` (Participants: Dave Jones)` will be auto appended, we probably don't need to repeat it in each piece of hidden text.

File: server/routes/journeys/record-review/participants-summary/view.njk (hunk 2 above)
Comment (ReedSoftware) [reply server/routes/journeys/record-review/participants-summary/view.njk:20]: It's in the types for them though?

File: server/routes/journeys/record-review/participants-summary/view.njk (hunk 3 above)
Comment (ReedSoftware) [reply server/routes/journeys/record-review/participants-summary/view.njk:22]: Oh? Does the component do that itself?

Comment (bill-moj) [reply server/routes/journeys/record-review/participants-summary/view.njk:22]: yea it's magic by the govuk component

File: server/routes/journeys/record-review/participants-summary/view.njk (hunk 2 above)
Comment (bill-moj) [reply server/routes/journeys/record-review/participants-summary/view.njk:20]: probably when we `record-review` we use `/:index`,
later on when we `update-review` we will use `/:attendeeUuid`.

i think we did the same with contributory factors and interviewees and identified needs.

Comment (ReedSoftware) [reply server/routes/journeys/record-review/participants-summary/view.njk:20]: If we have the uuid we may as well use it right? We talked about wanting to refactor the others to use it at some point.

Comment (ReedSoftware) [reply server/routes/journeys/record-review/participants-summary/view.njk:20]: @bill-moj ? :)

**Review by bill-moj — APPROVED** [review]

**Pull request merged**